File: Topic0102/013_check_n/check_n.js
const checkN = (n) => {
  let i = 2;

  while (i < n) {
    if (n % i == 0) return true;
    i++;
  }
  return false;
};

for (let i = 3; i < 20; i++) console.log(`${i} is not prime : ${checkN(i)}`);

File: Topic0102/013_check_n/check_n.java
class Check_N{
  static boolean checkN(int n){
    int i = 2;
    
    while(i < n){
      if(n % i == 0) return true;
      i++;
    }

    return false;
  }

  static public void main(String[] args){
    for(int i = 3; i < 20; i++)
      System.out.println(i + " is not prime : " + checkN(i));
  }
}
